HACKING.md

```diff
@@ -268,9 +268,14 @@ oc create secret generic slack-api-token --from-literal=text="${TOKEN}"
 oc label secret/slack-api-token \
     jenkins.io/credentials-type=secretText
 oc annotate secret/slack-api-token \
-    jenkins.io/credentials-description="Slack API token"
+    jenkins.io/credentials-description="Slack API token" \
+    jenkins.io/default-channel="#fedora-coreos-jenkins"
```

Comment:
don't know if you were trying to be accurate or just giving an example here but the actual channel we use is `#jenkins-fedora-coreos`

Reply:
actually it might be better her if we just make it generic and drop the paragraph below (the actual values will be in bitwarden anyway).

```
TOKEN=<token>
CHANNEL=<defaultChannel>
...
```

Reply:
Made into a variable. I'll update the BitWarden secrets once this merges.